README.md

```diff
@@ -1,13 +1,19 @@
 ## Celo Blockchain
 
-Official golang implementation of the Celo Blockchain, based off of the [official golang implementation of the Ethereum protocol](https://github.com/ethereum/go-ethereum).
+Official golang implementation of the Celo blockchain, based off of the [official golang implementation of the Ethereum protocol](https://github.com/ethereum/go-ethereum).
 
-## Building the source
+[![Discord](https://img.shields.io/badge/discord-join%20chat-blue.svg)](https://discord.gg/6yWMkgM)
+
+Prebuilt [Docker](https://en.wikipedia.org/wiki/Docker_\(software\)) images are available for immediate use: [us.gcr.io/celo-testnet/celo-node](https://us.gcr.io/celo-testnet/celo-node). See [docs.celo.org/getting-started](https://docs.celo.org/getting-started) for a guide to the Celo networks and how to get started.
+
+Documentation for Celo more generally can be found at [docs.celo.org](https://docs.celo.org/)
 
-Building Celo requires both a Go (version 1.9 or later), a C compiler and a recent nigthly Rust toolchain.
-You can install them using your favourite package manager. You may also consult the relevant sections in the [Celo Engineering Setup](https://github.com/celo-org/celo-monorepo/blob/master/SETUP.md) guide.
+Most functionality of this client is simmilar to `geth`, from which it was forked, if you do not find your question answered by Celo-specific documentation, try searching the [geth wiki](https://github.com/ethereum/go-ethereum/wiki).
 
-Once the dependencies are installed, run
+## Building the source
+
+Building `geth` requires both a Go (version 1.13 or later) and a C compiler and the latest Rust toolchain.
```

Comment:
There's an issue which causes a test to fail on 1.14 until we merge the trivial upstream fix for it.
```suggestion
Building `geth` requires both a Go (version 1.13) and a C compiler and the latest Rust toolchain.
```

Reply:
I thought Kobi removed the need for the Rust toolchain? not sure on progress of this..

Reply:
I just tested this by running a clean build on `master` and indeed the Rust compilation step no longer runs!